content/electrical/dc.md

```diff
@@ -0,0 +1,452 @@
+---
+jupyter:
```

Comment:
I don't know why this line is here, it doesn't show up for the other tutorials. Please check that this header is the same as for the other documents, so we don't have problems with jupytext and mystnb/sphinx.